io_scene_godot/export_godot.py

```diff
@@ -156,6 +156,10 @@ def export_object(self, obj, parent_gd_node):
 
     def should_export_object(self, obj):
         """Checks if a node should be exported:"""
+        for dimension in obj.scale:
```

Comment:
Just for your information, looks like scale with all dimensions being negative is valid in Godot (https://github.com/godotengine/godot/issues/26173), but with some limitations.
I am okay with your implementation by skipping any negative scale object.